endpoints/schemas/ordersSchemas.yaml

```diff
@@ -50,4 +50,22 @@ components:
         limitCount: 
           description: Max number of items to return
           type: integer #TODO any constraints on this?
-          format: int32
\ No newline at end of file
+          format: int32
+
+    ordersSubscriptionRequest:
+      description: A client creates a subscription by issuing a request against the subscriptions resource. The subscription namespace is client-defined via the POST operation.
+      required:
+        - resource
+        - notificationUrl
+      properties:
+        resource:
+          description: Resource path to watch
+          type: string
+          enum: [AcknowledgeReceiptNotification, CancelOrderResponse, CancelOrderResponse, ConfirmOrderNotification, CreateOrderResponse, OpenOrderNotification, ReviseOrderResponse]
+        notificationUrl:
```

Comment:
maybe it's not related to this PR, but we need to figure out a way to specify authentication information for these endpoints

Reply:
All the endpoints use the OAS require bearer definition, which doesn’t describe the properties of the jwt but does specify that one is required with the request: https://swagger.io/docs/specification/authentication/bearer-authentication/

We will also be adding the OAS definition for the authentication endpoint